src/win/fs.c

```diff
@@ -1403,13 +1404,18 @@ static void fs__fchmod(uv_fs_t* req) {
 }
 
 
-INLINE static int fs__utime_handle(HANDLE handle, double atime, double mtime) {
-  FILETIME filetime_a, filetime_m;
+INLINE static int fs__utime_handle(HANDLE handle, double btime, double atime, double mtime) {
+  FILETIME filetime_b, filetime_a, filetime_m;
+  int btime_provided = !isnan(btime);
+
+  if (btime_provided) {
+    TIME_T_TO_FILETIME((time_t) btime, &filetime_b);
```

Comment:
> Does the cast to time_t mean there is no sub-second precision on Windows?

The cast is a bit strange actually ... the unit of time measurement on NT(FS) is 100ns so we should be able to support subsecond precision.